### PR TITLE
Disable ground-truth availability checks

### PR DESCRIPTION
**Pull request by jimchamp**
Disables ground-truth availability calls for book page main borrow buttons.  Adds additional profiling to LoanStatus.html.

### Stakeholders

@mekarpeles

### CODE REVIEW

**Review by mekarpeles — APPROVED** [review]
lgtm

**Comment by jimchamp** [comment]
I would like to get some baseline metrics for borrow failures before this gets merged.  Converting to draft until we discuss the plan.

File: openlibrary/macros/databarWork.html [hunk 1]
```diff
@@ -51,8 +51,7 @@ <h3 class="header">
         $ edition = page.works and page
         $ lists_widget = render_template('lists/widget', edition or work, include_rating=True, include_header=False, include_widget=True, include_showcase=False, user_lists_only=False)
 
-        $# For the Works & Editions page, always use ground_truth_availability API for selected Edition
-        $:macros.LoanStatus(page, allow_expensive_availability_check=True, secondary_action=editions_page, check_sponsorship=editions_page, sponsorship_help=editions_page, check_loan_status=editions_page, post=lists_widget)
+        $:macros.LoanStatus(page, secondary_action=editions_page, check_sponsorship=editions_page, sponsorship_help=editions_page, check_loan_status=editions_page, post=lists_widget)
```
Comment (mekarpeles) [inline openlibrary/macros/databarWork.html:54]: I think we may want to adapt this so if the `page.availability["status"]` if `error` or `private` we still do the more expensive check. This shouldn't affect most books and will result in the correct result being shown in the cases which are most likely to be wrong from Bulk Availability API.

Comment (mekarpeles) [reply openlibrary/macros/databarWork.html]: I think this should be resolved w/ the latest change I made

File: openlibrary/macros/databarWork.html [hunk 2]
```diff
@@ -55,9 +55,11 @@ <h3 class="header">
         $ lists_widget = render_template('lists/widget', edition or work, include_rating=True, include_header=False, include_widget=True, include_showcase=False, user_lists_only=False)
         $ render_times['databarWork: List widget'] = time() - render_times['databarWork: List widget']
 
-        $# For the Works & Editions page, always use ground_truth_availability API for selected Edition
+        $# For the Works & Editions page, only use ground_truth_availability API for selected Edition
+        $# if edition availability status is private or error
```
Comment (mekarpeles) [inline openlibrary/macros/databarWork.html:59]:
```suggestion
        $# For the Works & Editions page, only use ground_truth_availability API for selected Edition
        $# if edition availability status is private or error
```

**Comment by mekarpeles** [comment]
Note: on http://ol-dev1.us.archive.org:1337/ (not testing -- I think there may have been a conflict w/ branches on testing)

**Comment by mekarpeles** [comment]
This successfully and appropriately hits the `Ground-Truth API`. Takes ~4.5s and .7s is ground truth.
http://ol-dev1.us.archive.org:1337/works/OL51940W/Meno?debug=true

This successfully uses `Bulk Availability API` and takes 1.6s and .25 is databar.
http://ol-dev1.us.archive.org:1337/books/OL23319049M/Meno?debug=true

Another example here ~1.5s
http://ol-dev1.us.archive.org:1337/books/OL5944019M/New_challenges_in_the_social_studies?debug=true

**Pull request merged**